swagger.yml

```diff
@@ -177,6 +177,257 @@ paths:
             $ref: '#/definitions/sniCertificate'
         default:
           $ref: '#/responses/error'
+  /accounts/{account_id}/env:
```

Comment:
I'm not an expert on how we treat environment variables but does everyone in a team have access all environment variables from a team?

Reply:
Collaborators don't have access to team env vars in the UI. I could be mistaken, but I believe Collaborators have read-only access via CLI/API (so that `netlify dev` can still build the site locally). Collaborators have read/write access to all site-level env vars.